docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml

```diff
@@ -964,6 +964,16 @@
 
               <entry>If true, enables ASSERT checking.</entry>
             </row>
+
+            <row>
+              <entry><emphasis>soapTraceLevel</emphasis></entry>
+
+              <entry>Default: 1</entry>
+
+              <entry>The level of detail in reporting SOAPCALL information
```

Comment:
This actually affects HTTPCALL as well... the option name is historical from before HTTPCALL existed.